clayui.com/svg-icons.js

```diff
@@ -670,6 +670,7 @@ const iconsData = {
 	"rss":						'"notification", "updates"',
 	"rules":					'"condition"',
 	"search":					'"inspect", "lens", "zoom"',
+	"search-plus":				'"inspect", "lens", "zoom"',
 	"select":					'"input", "multiple", "options"',
```

Comment:
Whats with the whitespacing in this file? Can we normalize that at all?

Reply:
@bryceosterhaus I used tabs to line up the aliases, not sure why the formatting on github is off. I bet if I switch it to spaces github would line them up correctly.